src/Avalonia.X11/X11Window.cs

```diff
@@ -759,7 +760,7 @@ private void ScheduleInput(RawInputEventArgs args)
 
         private void MouseEvent(RawPointerEventType type, ref XEvent ev, XModifierMask mods)
         {
-            if (_inputRoot is null)
+            if (_inputRoot is null || _disabled)
```

Comment:
`MouseEvent` method is only used on XInput-incapable systems. If you want to prevent certain events from being raised, add a check to `DispatchInput`. We probably _don't_ want to filter events like "pointer capture lost" or enter/leave events.